core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -1481,6 +1643,46 @@
             comment="Magnetorquers operating and automatically suspended."
             nvalue="3" value="SUSPEND" />
         </mal:enumeration>
+        <mal:enumeration
+          comment="Wheel identifiers."
+          name="ReactionWheelIdentifier" shortFormPart="12">
+          <mal:item comment="Reaction wheel x"
+                    nvalue="1" value="WHEEL_X" />
+          <mal:item comment="Reaction wheel y"
+                    nvalue="2" value="WHEEL_Y" />
+          <mal:item comment="Reaction wheel Z"
+                    nvalue="3" value="WHEEL_Z" />
+          <mal:item comment="Reaction wheel u"
+                    nvalue="4" value="WHEEL_U" />
+          <mal:item comment="Reaction wheel v"
+                    nvalue="5" value="WHEEL_V" />
+          <mal:item comment="Reaction wheel w"
+                    nvalue="6" value="WHEEL_W" />
+        </mal:enumeration>
+        <mal:composite
+          comment="Holds Raction wheel parameters" name="ReactionWheelParameters"
+          shortFormPart="12">
+          <mal:field canBeNull="false"
+                     comment="Controll mode of the Reaction wheels. 0 = Target value is a current value [A]; 1 = Target value is a speed value [rad/sec]; 2 = Target alue is a torgh [Nm]" name="controlMode">
```

Comment:
s/Controll/Control/

Reply:
Thanks!
I just fixed it. (I also rebased onto current dev branch.)